it/mail/chrome/messenger/preferences/compose.dtd

```diff
@@ -118,28 +118,28 @@
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbCharactersGoal.label "Character Count">
-<!ENTITY pbStopwatchGoal.label "Stopwatch">
-<!ENTITY addSignatureDelimiters.label "Add delimiter above the signature">
-<!ENTITY pbSetCompositionGoals.label "Set composition goals using">
-<!ENTITY pbWordsGoal.label "Word Count">
-<!ENTITY pbMinutes.label "minutes">
-<!ENTITY pbAt.label "at">
-<!ENTITY sourceView.label "Code View:">
-<!ENTITY pbWords.label "words">
+<!ENTITY pbCharactersGoal.label "Conteggio caratteri">
+<!ENTITY pbStopwatchGoal.label "Cronometro">
+<!ENTITY addSignatureDelimiters.label "Aggiungi un delimitatore sopra la firma">
+<!ENTITY pbSetCompositionGoals.label "Imposta obiettivi di composizione utilizzando">
+<!ENTITY pbWordsGoal.label "Conteggio parole">
+<!ENTITY pbMinutes.label "minuti">
+<!ENTITY pbAt.label "a">
+<!ENTITY sourceView.label "Visualizzazione codice:">
+<!ENTITY pbWords.label "parole">
 <!ENTITY sourceView.accesskey "C">
-<!ENTITY pbCharacters.label "characters">
+<!ENTITY pbCharacters.label "caratteri">
 
-<!ENTITY pbURLAutoComplete.label "Enable URL Autocomplete">
+<!ENTITY pbURLAutoComplete.label "Abilita autocompletamento URL">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY messageSizeWarning.label "Warn me if the message size is above">
+<!ENTITY messageSizeWarning.label "Avvisami se le dimensioni del messaggio sono sopra">
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+<!ENTITY messageSizeWarning.label "Avvisami se le dimensioni del messaggio superano">